### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized Control Type property is wrong for the "Scroll Left" and "Scroll Right" buttons

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#52]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://marketplace.visualstudio.com/VSCode](https://marketplace.visualstudio.com/VSCode) in EDGE
2. Navigate to the "Scroll Left" and "Scroll Right" buttons available to the left and right of the Extensions link under "Most Popular", "Trending" and "Recently Added" extension headers.
3. Observe the "Localized Control Type" property in Inspect

**Actual Result:**
The "Localized Control Type" property is "link"
**_Narrator Behaviour:_**
Narrator announces as "Scroll Left Link" and "Scroll Right Link"

**Expected Result:**
The "Localized Control type" property should be "Button"
**_Eg: "Scroll Left Button" and "Scroll Right Button"_**

**User Impact:**
Visually impaired will be confused about the control type of the scroll left and scroll right buttons and they will consider that clicking on the buttons will redirect to some other page
 
**Screenshots:**
![localized control type of scroll left button is link](https://cloud.githubusercontent.com/assets/24825441/21718594/6d85f17a-d43e-11e6-8926-c1cba3e6bfdb.png)
![localized control type of scroll right button is link](https://cloud.githubusercontent.com/assets/24825441/21718593/6d859b76-d43e-11e6-9a2a-30e19ff2eea1.png)

**Comment by waderyan** [comment]
Marketplace
